docs/design/generics/details.md

```diff
@@ -3763,8 +3784,8 @@ interface when its element type satisfies the same interface:
     if the element type is comparable.
 -   A container is copyable if its elements are.
 
-To do this with an [out-of-line `impl`](#external-impl), specify a more-specific
-`Self` type to the left of the `as` in the declaration:
+This may be done with an [external `impl`](#external-impl) by specifying a
```

Comment:
As I'm glancing at this again -- a later example clarifies it's intended to work, so maybe just adjust the example to show inline?

Reply:
- I made the change to make it more accurate, inline external `impl` is supported for this.
- I will delete the word `Self` since it seems to be causing some confusion. It is true that you can't put `Self` before `as`, but I was referring to it as the `Self` type since `Self` inside the `impl` definition body is set to that type.
- External impls can be declared inline, but they will most commonly be out-of-line, and the example reflected that. There is an example further down to show it can also be inline.

Reply:
Thanks! I agree removing `Self` clarifies.